.github/workflows/verify-formatting.yml

```diff
@@ -0,0 +1,14 @@
+name: "Verify code formatting"
+
+on:
+  - push
+  - pull_request
+
+jobs:
+  stylecheck:
+    steps:
+      - name: "Checkout code"
+        uses: actions/checkout@v2
+
+      - name: "Verify Markdown formatting"
+        run: npx prettier@2.0.4 --check "**/*.md"
```

Comment:
Should this check everything or only the files affected by the PR/push unless on master?

Reply:
Good point. I'll try and have it only check the relevant files